docs/sources/administration/home_dashboard.md

```diff
@@ -0,0 +1,36 @@
++++
+title = "How to replace the default home dashboard"
+description = "How to replace the default home dashboard"
+keywords = ["grafana", "configuration", "documentation", "home"]
+type = "docs"
+[menu.docs]
+name = "Home Dashboard"
+identifier = "home-dashboard"
+parent = "admin"
+weight = 100
++++
+
+# How to replace the default home dashboard
+
+You can easily replace the default homepage of Grafana in two different ways.
+
+## Preferences
+
+You can choose any of your existing Grafana dashboards to replace the default home dashboard in the preferences. First, you need to star the dashboard that you want to use. Then, in `Configuration > Preferences`, you can select one of your starred dashboards in the `Home Dashboard` input. It will set it for all the users in your organization.
+
+You can also replace it for you only by updating your own preferences.
+
+If you change the home dashboard in both preferences, user preferences will override organization preferences. 
+
+
+## Configuration
+
+You can also provide your own JSON file and set the path to this file in the [configuration]({{< relref "./configuration.md">}}) file.
```

Comment:
```suggestion
You can provide your own JSON file and set the path to this file in the [configuration]({{< relref "./configuration.md">}}) file.
```

Reply:
I recommend linking to the specific heading with the setting they need to update.

Reply:
Also, be sure to mention that this should be in their custom.ini, not their default.ini.

Reply:
I think it's best to keep the generic link here because it's for people that don't know how to update their configuration file (and I don't want to repeat all the ways of doing so here). There isn't more information on the setting in the configuration documentation than in this file. Also for adding the mention to custom.ini, I'm not sure. It's grafana.ini on other OS and I think if the user doesn't know this, he should read the documentation on configuration first.

Maybe I should just rephrase it with something like: `Update your configuration file to set the path to the JSON file. Read how to update this file in the [configuration]({{< relref "./configuration.md">}}) documentation`?

Reply:
Yes, I like that change. You're right, it will help address users of varying skill levels.